packages/fe-fpm-writer/src/section/types.ts

```diff
@@ -0,0 +1,46 @@
+import { CustomElement, InternalCustomElement, Position } from '../common/types';
+
+/**
+ * Enumeration with all values possible for custom section type
+ * Currently only xml fragment is supported.
+ */
+export enum CustomSectionType {
+    XMLFragment = 'XMLFragment'
+}
+
+export interface CustomSection extends CustomElement {
+    /**
+     * Name of the routing target
+     */
+    target: string;
+
+    /**
+     * Extension file type. Currently only xml fragment is supported.
+     */
+    type: CustomSectionType;
```

Comment:
Do we plan to support to anything else here in the future (in the Fiori tools)? If not, I would consider removing it or at least making it optional and then using `XMLFragment` if the nothing is provided by the caller.

Reply:
@tobiasqueck 
According to latest information from Fiori team(from email conversation). According to Fiori team statement - there was plan at beginning to use `type` and add additional type like `XMLView`, but according to latest statement team decided to stick with fragments. Also support for `JSFragments` not planned as well.
Proposal from Fiori team was that we ignore such property.

I deleted `type` property in commit 1a331770d57845ac7aa54cd89b172837f242a64d